src/Sylius/Bundle/ApiBundle/Converter/CommandFieldItemIriToIdentifierStorageInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Converter;
+
+interface CommandFieldItemIriToIdentifierStorageInterface
```

Comment:
As internal storage for data, I don't think we need an interface for this data structure.